compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -368,6 +379,18 @@ public BType resolvePatternTypeFromMatchExpr(BLangExpression matchExpr, BLangExp
         return symTable.noType;
     }
 
+    public BType resolvePatternTypeFromMatchExpr(BLangMappingMatchPattern mappingMatchPattern, BType patternType,
```

Comment:
Please recheck the access modifiers for these methods. Seems like they don't have to be public.

Reply:
Fixed